frameworks/PHP/simps/app/Model/DbModel.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * This file is part of Simps.
+ *
+ * @link     https://simps.io
+ * @document https://doc.simps.io
+ * @license  https://github.com/simple-swoole/simps/blob/master/LICENSE
+ */
+
+namespace App\Model;
+
+use Simps\DB\BaseModel;
+
+class DbModel extends BaseModel
+{
+    public function fortunes()
+    {
+        $fortune = [];
+        $this->pdo->fortune_test = $this->pdo->fortune_test ?? $this->pdo->prepare('SELECT id, message FROM Fortune');
```

Comment:
For me that don't look like a ful ORM,
You are using the PDO directly and with SQL.

Micro ORM at most.

Reply:
I used the pdo connection pool and did not use orm. How should it be modified?

Reply:
I want to add a separate ORM pr in the future.